package.json

```diff
@@ -23,7 +23,21 @@
     },
     "devDependencies": {
         "dtslint": "latest",
+        "husky": ">=1",
+        "lint-staged": ">=8",
+        "prettier": "^1.17.1",
         "types-publisher": "github:Microsoft/types-publisher#production"
     },
-    "dependencies": {}
+    "dependencies": {},
+    "husky": {
```

Comment:
It's quite easy to setup in a new project:
```
git init
yarn init
yarn add -D prettier
npx mrm lint-staged
```
Then make some changes and commit.

Reply:
Regarding Prettier, it's fairly standard now in new codebases. If you'd like to give that a try and see how it works as you write code, install the Prettier VSCode extension; it will tap into the `prettierrc` file and as you safe will apply. It's a wonderful energy saver and I encourage checking out [the website](https://prettier.io/). Otherwise, it will format on pre-commit.